components/script/dom/document.rs

```diff
@@ -2159,6 +2161,25 @@ impl Document {
         self.process_deferred_scripts();
     }
 
+    #[allow(unsafe_code)]
+    pub fn deferred_module_script_loaded(&self, element: &HTMLScriptElement, url: ServoUrl) {
```

Comment:
Default behavior for module script is `defer` so I named it with the `deferred` like `classic script`.

I created this function because I'd like to call a different loaded function when I'd like to finish load for modules.

But, I found I can distinguish if the script is module or not with the original `deferred` loaded function so I removed it in https://github.com/servo/servo/pull/23545/commits/5e636fc23376d5289dddb8beb52782042334c7f1 🙏